snuba/datasets/errors_replacer.py

```diff
@@ -21,24 +21,27 @@
     Union,
 )
 
-from snuba import settings
+from snuba import environment, settings
 from snuba.clickhouse import DATETIME_FORMAT
 from snuba.clickhouse.columns import FlattenedColumn, Nullable, ReadOnly
 from snuba.clickhouse.escaping import escape_identifier, escape_string
+from snuba.datasets.events_processor_base import REPLACEMENT_EVENT_TYPES
 from snuba.datasets.schemas.tables import WritableTableSchema
 from snuba.processor import InvalidMessageType, _hashify
 from snuba.redis import redis_client
 from snuba.replacers.replacer_processor import Replacement as ReplacementBase
 from snuba.replacers.replacer_processor import ReplacementMessage, ReplacerProcessor
-
-logger = logging.getLogger(__name__)
+from snuba.utils.metrics.wrapper import MetricsWrapper
 
 """
 Disambiguate the dataset/storage when there are multiple tables representing errors
 that perform event replacements.
 In theory this will be needed only during the events to errors migration.
 """
 
+logger = logging.getLogger(__name__)
```

Comment:
Why do you need this ?

Reply:
This just got moved in my PR. Nothing changed functionally.